flink-kubernetes/src/main/java/org/apache/flink/kubernetes/utils/KubernetesUtils.java

```diff
@@ -288,6 +290,21 @@ public static ResourceRequirements getResourceRequirements(int mem, double cpu)
 			.build();
 	}
 
+	/**
+	 * Parses comma-separated list of imagePullSecrets into K8s-understandable format.
+	 */
+	public static LocalObjectReference[] parseImagePullSecrets(@Nullable String imagePullSecrets) {
```

Comment:
IIRC the community introduce a list type config option. You can refer to `PipelineOptions.JARS` for an example.

Reply:
Good catch. Will try it out.